CHANGELOG.md

```diff
@@ -1,3 +1,52 @@
+# 107.0.0
+
+In this major release we deleted the deprecated themer classes: `MDCAlertScheme` and `MDCAlertControllerThemer`.
+
+## Breaking changes
+
+Deleted `MDCAlertScheme` and `MDCAlertControllerThemer` class, please use `MDCAlertController (MaterialTheming)` extension instead.
+
+## New features
+
+The new delegate MDCAlertControllerDelegate protocol exposes dialogs life cycle events that are useful for impression logging."
+In this release we added a new `delegate` property to `MDCAlertController` which exposes dialogs lifecycle events that are useful for impression logging. As well as various minor improvements to the `Dialogs` component and other bug fixes.
```

Comment:
This sentence seems partially redundant. What about this:
>`MDCAlertController` now has a `delegate` property conforming to the new `MDCAlertControllerDelegate` property.

Reply:
Forgot to delete the first sentence sorry.